keyboards/viktus/sp111_v2/keymaps/via/keymap.c

```diff
@@ -0,0 +1,252 @@
+    KC_P1,  KC_P2,  KC_P3,  KC_PPLS,    KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(1),            KC_UP,
+    KC_P0,  KC_P0,  KC_PDOT,  KC_PENT,  KC_LCTL, KC_LGUI, KC_LALT, KC_MUTE, KC_SPC,           KC_SPC,  KC_APP,  KC_RALT, KC_RGUI, KC_RCTL,                            KC_LEFT, KC_DOWN, KC_RIGHT),
+
+    [1] = LAYOUT(
+    _______, _______, _______, _______,      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, QK_BOOT,
+    _______, _______, _______, _______,      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______,      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
+    _______, _______, _______, _______,      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
+    _______, _______, _______, _______,      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+    _______, _______, _______, _______,      _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______,                            _______, _______, _______),
+
+    [2] = LAYOUT(
+    _______, _______, _______, _______,      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, QK_BOOT,
+    _______, _______, _______, _______,      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______,      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
+    _______, _______, _______, _______,      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
+    _______, _______, _______, _______,      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+    _______, _______, _______, _______,      _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______,                            _______, _______, _______)
+};
+
+#ifdef RGB_MATRIX_ENABLE
```

Comment:
Unfortunately, custom code for indicators is no longer being accepted to the repo and will have to be removed for the PR to be accepted.

Reply:
I've removed. Thanks